corehq/apps/reports/static/reports/js/user_history.js

```diff
@@ -4,7 +4,7 @@ hqDefine("reports/js/user_history", [
     $
 ) {
     $(function () {
-        if (document.location.href.match(/reports\/user_history/)) {
+        if (document.location.href.match(/reports\/user_management\/user_history/)) {
```

Comment:
strange, how was that working earlier?

Reply:
Good question, it was definitely working before. I looked for where/when it might have been changed recently, but haven't found anything at this point.